src/generators/css-filter.ts

```diff
@@ -79,16 +78,7 @@ export function cssFilterStyleSheetTemplate(filterValue: string, reverseFilterVa
 
     if (!frameURL) {
         // If user has the chrome issue the colors should be the other way around as of the rootcolors will affect the whole background color of the page
-        const rootColors = hasChromiumIssue501582() && config.mode === FilterMode.dark ? [0, 0, 0] : [255, 255, 255];
-        const [r, g, b] = applyColorMatrix(rootColors, createFilterMatrix(config));
-        const bgColor = {
-            r: Math.round(r),
-            g: Math.round(g),
-            b: Math.round(b),
-            toString() {
-                return `rgb(${this.r},${this.g},${this.b})`;
-            },
-        };
+        const bgColor = hasChromiumIssue501582() && config.mode === FilterMode.dark ? 'black' : 'white';
```

Comment:
I don't recall anything on why we needed colorMatrix to be applied over these values. It doesn't seem to matter?

@alexanderby could you enlighten this a bit?

Reply:
The issue is that after some Chrome version the filter is not applied to the transparent background. So we add a color to make it not transparent. Then the user wants to apply his preferred brightness, contrast and sepia to this color.

Reply:
I think I guess, the filter used to not apply to the background of root element at all, even if some color was not assigned, maybe they've changed it in a way that it is now applied.

Reply:
My understanding of the issue was wrong last year.

So, if there is an issue 501582 (version before 81), the old logic should work. If not, just put a white color, it should be inverted by filter in dark mode. For ancient browsers we just don't care.